widgetssdk/src/main/res/layout/visitor_video_floating_view.xml

```diff
@@ -12,4 +13,17 @@
         android:text="@string/glia_call_visitor_video_on_hold_text"
         android:textColor="@color/glia_base_light_color" />
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
I'd suggest using `ImageButton` instead of `FloatingActionButton`, since `FloatingActionButton` is part of material design.

Reply:
We already use FloatingActionButtons on the call screen. Do you want to replace them all or just this button?

Reply:
I am confident that we will eventually eliminate the material theme. Moreover, I prefer to avoid adding new dependencies to it to simplify the migration process.

Reply:
fixed